editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -58,11 +77,50 @@ function MapAndLabelComponent(props: Props) {
                 name="fn"
                 placeholder={"Data Field"}
                 value={formik.values.fn}
+                onChange={formik.handleChange}
                 required
               />
             </InputRow>
           </InputGroup>
         </ModalSectionContent>
+        <ModalSectionContent title="Map Formatting">
+          <InputGroup>
+            <InputRow>
+              <InputRowItem>
+                <ColorPicker
+                  label="Drawing Colour"
+                  color={formik.values.drawColour}
+                  onChange={(color) => {
+                    formik.setFieldValue("drawColour", color);
+                  }}
+                  errorMessage={formik.errors.drawColour}
+                />
+              </InputRowItem>
+            </InputRow>
+          </InputGroup>
+        </ModalSectionContent>
+        <ModalSectionContent title="Map Drawing Type">
```

Comment:
nit: I think we want `Map drawing type` to be associated as an input label to the radio group rather than a modal section content header?